pkg/rc/replication_controller.go

```diff
@@ -35,9 +36,9 @@ type ReplicationController interface {
 	// The error channel will be closed in response.
 	WatchDesires(quit <-chan struct{}) <-chan error
 
-	// CurrentNodes() returns all nodes that this replication controller is currently scheduled on,
+	// CurrentPods() returns all nodes that this replication controller is currently scheduled on,
 	// by selecting pods labeled with the replication controller's ID.
-	CurrentNodes() ([]string, error)
+	CurrentPods() (map[string]pods.Manifest, error)
```

Comment:
`CurrentPods()` is like `CurrentNodes()`, but it also looks up the nodes' intent manifests. After looking a bit more at this, I think it makes sense to keep `CurrentNodes()` in the public interface. Fetching the manifest involves more network operations, so I'd rather avoid them if possible, and it turns out that many users of `CurrentPods()` don't care about the manifest.